pkg/test/suite/run.go

```diff
@@ -2,6 +2,7 @@ package suite
 
 import (
 	"fmt"
+	"github.com/justtrackio/gosoline/pkg/dispatcher"
```

Comment:
This import line should go down to the others.. :)

Reply:
done